app/assets/javascripts/controllers/credentials/credentials_controller.js

```diff
@@ -48,6 +48,10 @@ ManageIQ.angular.app.controller('credentialsController', ['$http', '$scope', 'mi
     return $scope.newRecord || (!$scope.showChangePasswordLinks(userid)) || $scope.bChangeStoredPassword;
   };
 
+  $scope.showValidate = function(tab) {
+    return !($scope.emsCommonModel.emstype == 'openstack_infra' && $scope.newRecord && tab == 'ssh_keypair')
```

Comment:
@petrblaho The angular `credentialsController` is designed to be a very generic controller, meaning it should not get into the specifics of the model in question. 
All model specific changes should be done in the angular controller that creates the model. 
You should move this method to `emsCommonFormController`